.github/workflows/render.yml

```diff
@@ -0,0 +1,21 @@
+name: Render pdfs
+
+on:
+  pull_request:
+    branches: [main]
```

Comment:
Does this mean that any time a commit is pushed to an open PR, a second commit will automatically be pushed with the PDF update? That seems like it might lead to unnecessary bloat of the git repo (which increases by ~3MiB every regeneration because git doesn't do binary diffs well) as suggestions are applied and comments responded to.

If we want to reduce the frequency of rebuilds while always ensuring https://zips.z.cash/ is up to date:
```suggestion
on:
  push:
    branches:
      - main
    paths:
      - 'protocol/**'
```